tools/c7n_openstack/tests/data/flights/default.yaml

```diff
@@ -217,4 +217,17 @@ son"}]}]}}'}
     headers:
       Content-Type: [test/html]
     status: {code: 200, message: OK}
```

Comment:
I updated the tests.
I suppose there are a few problems with the test script. Because first of all, the recorded files don't have file extension. Secondly, to record new resources, I had to comment the following line, because I have different URLs for OpenStack.
https://github.com/cloud-custodian/cloud-custodian/blob/35056d5ae90dc479e0f36821405a0334a85250d3/tools/c7n_openstack/tests/common_openstack.py#L38 

I don’t fully understand the code of how the tests work, and I can’t figure out how to fix it.